### DISCUSSION TITLE
CTOV Compat

### DISCUSSION THREAD

**Discussion by femanmk42** [ejektaflex/Bountiful#198] [Mod Compat]
I cannot find bounty boards in ctov villages, even with the mod turned all the way up they just do not spawn. Therefore, compatibility would be appreciated.

**Comment by ejektaflex** [comment]
What is CTOV? What modloader is this for?

**Comment by femanmk42** [comment]
> What is CTOV? What modloader is this for?

CTOV is choice theorems overhauled village. It is for the fabric modloader, sorry for confusion.

**Reply by vizthex123** [reply]
it's also on Forge too. I have it in my new forge modpack.

**Comment by dowttie** [comment]
I'm experiencing the same atm on 1.19.2
Edit: found the datapack for it, sorry for report! 
https://modrinth.com/datapack/ctov-bountiful-compat

**Comment by ChoiceTheorem** [comment]
If you need built-in compatibility, we can talk here. What I'm working on should make even that datapack optional.

**Reply by ChoiceTheorem** [reply]
BTW, I already put up a beta version of built-in compatibility between CTOV and Bountiful but I suggest you implement different variants of the bounty board that TG made a pull request, It would help me a lot.

**Comment by ejektaflex** [comment]
Hi @ChoiceTheorem - yeah, I could choose to not inject the jigsaw pieces in the case that your mod is loaded. If you need anything on my end, let me know and I'll see what I can do (though it won't be overly extensive 🙃 )!

**Comment by ChoiceTheorem** [comment]
My current idea is to spawn a bounty board pieces in the town center. The problem is it's not 100% reliable like using a datapack.

**Comment by ChoiceTheorem** [comment]
Or another way to do it is for you to take extra two nbt files. Everything else is on my end. Your call between these two ideas.

**Comment by ejektaflex** [comment]
Due to the nature of how Jigsaws work, I'm not sure if it's ever going to reliably add a board to every village. However, if you're referring to the center piece that the town starts as, that's a good idea! It would be a lot of fun to create a town center piece for the bounty board. I'll look into that.
